src/commands/account/import_account/mod.rs

```diff
@@ -60,6 +60,15 @@ pub fn login(
             ));
         }
         if access_key_view.is_err() {
+            // If the implicit account does not exist on the network, it will still be imported.
+            if crate::types::account_id::is_implicit(account_id_from_cli.as_ref()) {
+                let pk_implicit_account =
+                    near_crypto::PublicKey::from_near_implicit_account(&account_id_from_cli)?;
+                if public_key_str == pk_implicit_account.to_string() {
+                    break account_id_from_cli;
+                }
+            };
```

Comment:
We don't even need to query the RPC for the implicit account id. Move it up before calling `verify_account_access_key`.